File: WebClient/src/models/view-data/person-request-view-data.ts
// класс для представления данных по обращению человека в сервис
export class PersonRequestViewData{
  // конструктор со свойствами класса
  constructor(
    public id: number, // id - обращения
    public name: string, // имя персоны
    public srname: string, // фамилия персоны
    public patronymic: string, // отчество персоны
    public telephone: string, // телефон персоны
    public descriptionOfTheProblem: string, // описание проблемы
    public status: string // статус обращения
  ){}
}

File: WebApplication/WebApplication/Models/ViewData/PersonRequestViewData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Security.Cryptography;
using System.Threading.Tasks;
using Maintenance.Models.MaintenanceEntities;

namespace WebApplication.Models.ViewData
{
    public class PersonRequestViewData {
        // id
        public int Id { get; set; }
        // имя персоны
        public string Name { get; set; }
        // фамилия персоны
        public string Surname { get; set; }
        // отчество персоны
        public string Patronymic { get; set; }
        // паспорт персоны
        public string Passport { get; set; }
        // телефон персоны
        public string Telephone { get; set; }
        // описание проблемы
        public string DescriptionOfTheProblem { get; set; }
        // статус
        public string Status { get; set; }

        public PersonRequestViewData() { }
        public PersonRequestViewData(PersonRequest personRequest, Person person, PersonRequestStatus personRequestStatus) {
            Id = personRequest.Id;
            Name = person.Name;
            Surname = person.Surname;
            Patronymic = person.Patronymic;
            Passport = person.Passport;
            Telephone = personRequest.TelephoneNumber;
            DescriptionOfTheProblem = personRequest.DescriptionOfTheProblem;
            Status = personRequestStatus.Title;
        }
    }
}
